src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -359,6 +358,11 @@ private function createFirewall(ContainerBuilder $container, $id, $firewall, &$a
         // Access listener
         $listeners[] = new Reference('security.access_listener');
 
+        // Logout listener
+        if (isset($logoutListener)) {
```

Comment:
comment should be removed (low value)
the "isset" shold be replaced by `null !== $logoutListener` (which implies setting the var to null when appropriate)

Reply:
There is a comment like this above each listener. Should I remove all of them?

Reply:
Done